backend/app/Http/Controllers/MarketplaceController.php

```diff
@@ -74,7 +74,7 @@ public function putTransaction($card_id)
         $transaction = new Transaction();
         $transaction->card_id = $card_id;
         $transaction->user_id = $user->id;
```

Comment:
This was ok the way it was before.
```
$price = EthereumConverter::convertETHPriceToInt($listing->price);
```
Because we want the price to be stored as an Integer in the db